Lesson_29/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+services:
+  postgres:
+    image: postgres:latest
+    environment:
+      POSTGRES_DB: my_test_db
+      POSTGRES_USER: serhiiravliuk
+      POSTGRES_PASSWORD: kabosina2517
+    networks:
+      - mynetwork
+
+  app:
+    build: .
+    environment:
+      DATABASE_URL: postgres://serhiiravliuk:kabosina2517@postgres/my_test_db
+    depends_on:
+      - postgres
+    networks:
+      - mynetwork
+
+networks:
+  mynetwork:
```

Comment:
You do not specify network capabilities as I understand. So where is your db instance be hosted.